timelock-impl/src/test/java/com/palantir/atlasdb/timelock/lock/NamedMinTimestampLockCollectionTest.java

```diff
@@ -18,28 +18,83 @@
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
+import com.palantir.atlasdb.timelock.api.GenericNamedMinTimestamp;
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
+import java.util.Set;
 import java.util.UUID;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
 public final class NamedMinTimestampLockCollectionTest {
+    private static final UUID REQUEST_ID_1 = UUID.randomUUID();
+    private static final UUID REQUEST_ID_2 = UUID.randomUUID();
+    private static final UUID REQUEST_ID_3 = UUID.randomUUID();
+
     private final NamedMinTimestampLockCollection locks = new NamedMinTimestampLockCollection();
 
+    @MethodSource("getGenericNamedMinTimestamps")
+    @ParameterizedTest
+    public void namedMinTimestampLockDescriptorMatchesFormatForAllowedTimestampNames(
```

Comment:
I think the concept of an "allowed timestamp name" doesn't really exist now, other than maybe not overlapping with the immutable (and even then, that's enforced at the level of the name object rather than by this class right?)

Reply:
oopsies, missed clean up